src/resources/dataPoints/dataPointsApi.ts

```diff
@@ -35,7 +31,11 @@ export class DataPointsAPI {
    * const data = await client.datapoints.retrieve({ items: [{ id: 123 }] });
    * ```
    */
-  public retrieve: DataPointsRetrieveEndpoint;
+  public async retrieve(
+    query: DatapointsMultiQuery
+  ): Promise<(DatapointsGetAggregateDatapoint | DatapointsGetDatapoint)[]> {
```

Comment:
you sure it's not `DatapointsGetAggregateDatapoint[] | DatapointsGetDatapoint[]`?